### PR TITLE
Language direction

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/components/header.jsx [hunk 1]
```diff
@@ -55,12 +55,11 @@ function Header({ slug }) {
   );
 
   const handleClick = (langKey) => {
-    i18n.changeLanguage(langKey);
-    navigate(`/${langKey}/${slug}`);
+    i18n.changeLanguage(langKey, navigate(`/${langKey}/${slug}`));
```
Comment (saifulislamdev) [inline src/components/header.jsx:58]: Whaaaat, where'd you learn this?

File: src/styles/_class-overrides.scss [hunk 2]
```diff
@@ -299,14 +299,16 @@
     //position languages at the bottom
     position: absolute;
     bottom: 0;
+    left: 0;
```
Comment (saifulislamdev) [inline src/styles/_class-overrides.scss:302]: What is the order for your properties here?

File: src/components/header.jsx (hunk 1 above)
Comment (aannyyaa) [reply src/components/header.jsx:58]: I found it [here](https://www.i18next.com/overview/api#changelanguage) when I was looking into the language direction method

File: src/styles/_class-overrides.scss (hunk 2 above)
Comment (aannyyaa) [reply src/styles/_class-overrides.scss:302]: There isn't one for now - I can implement one in my sass organization PR

File: src/components/header.jsx (hunk 1 above)
Comment (saifulislamdev) [inline src/components/header.jsx:58]: Have you tried changing the language to see if this works?

Comment (aannyyaa) [reply src/components/header.jsx:58]: yes

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**